explainaboard/metric.py

```diff
@@ -334,18 +338,18 @@ def get_corpus_value(self) -> float:
         Return the evaluation metric value over all examples in the corpus.
         """
         self._fetch_results()
-        return self._corpus_value
+        return unwrap(self._corpus_value)
```

Comment:
This unwrap is probably not necessary, as `_fetch_results()` guarantees that it will be initialized. But it probably won't hurt to have it here.

Reply:
Sure, sounds good.